packages/react-components/react-card/src/components/Card/__snapshots__/Card.test.tsx.snap

```diff
@@ -4,7 +4,6 @@ exports[`Card renders a default state 1`] = `
 <div>
   <div
     class="fui-Card"
-    data-tabster="{\\"groupper\\":{}}"
```

Comment:
is the outcome of another tabster api change ?

Reply:
Not really of Tabster origin but rather an oversight of mine. I assumed a no parameter groupper usage would result in neutral functionality but, as it turns out, it doesn't.

Reply:
can we create issue on tabster repo to suggest to throw a warning in dev mode on this usecase ?

Reply:
Created an issue to get clarification: https://github.com/microsoft/tabster/issues/147